services/category.js

```diff
@@ -28,6 +28,7 @@ const categoryService = {
         }
       }
     ])
+    if (!minMaxPrices.length) minMaxPrices[0] = { min: 0, max: 0 }
```

Comment:
return object with same names. because you have minPrica,maxPrice, but if no prices you have min, max